### ISSUE TITLE
Bounce rate has jumped by 20% - any idea why?

### ISSUE THREAD

**Issue by valianttry** [mozilla/donate.mozilla.org#1017]
This is significant, and seemed to start on Friday. What changed??
![location_-_google_analytics](https://cloud.githubusercontent.com/assets/9830371/11524484/080a6948-9880-11e5-8dfb-e23064864923.png)

@adamlofting @jbuck @edrushka @cadecairos @alicoding

**Comment by adamlofting** [comment]
Updated the title as this applies to all locales, and all channels. It seems to most significantly impact the Snippet.

<img width="1168" alt="screenshot 2015-12-02 09 09 12" src="https://cloud.githubusercontent.com/assets/1901816/11526726/5e947ffc-98d4-11e5-8d4f-d5ffbb7ae4b0.png">

**Comment by adamlofting** [comment]
Panic averted :sweat_smile: 

This is just a stats thing and is having no impact on our fundraising (I'll explain).

**TLDR; Our conversion rate, and income are unnafected.**

**What changed?**
On the 27th we switched to the one-page form as our default.

**Why did this affect bounce rate?**
Bounce rate reflects how many people who land on a page do/don't click to another page on your site.

On the sequential form, if you choose an amount and move to the next page, you become a 'non-bounce' visit. Even if you then don't click the Credit Card or PayPal button. 

On the one-page form, the conversion funnel takes place on PayPal or Stripe overlay so doesn't show up in our bounce rate stats, because these page views are not on our site.

**Comment by jbuck** [comment]
@adamlofting is there anything we should add to the page to make the bounce rate a more accurate statistic?

**Comment by valianttry** [comment]
@adamlofting thanks for investigating -- as long as we are aware of the phenomenon, i don't think fixing it is a P1 @jbuck (agree @adamlofting ?)

**Comment by adamlofting** [comment]
> @adamlofting is there anything we should add to the page to make the bounce rate a more accurate statistic?

It depends on what you mean by accurate :wink: Technically, our new form bounces more users to another website. So it's 'accurate' in that sense.

I wouldn't invest any time in this right now, as the steps we're not tracking here are steps we can't influence (Stripe form and Paypal).
